one-ui/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.tsx

```diff
@@ -72,7 +72,7 @@ const SourceToEntityMap = (props) => {
 
     const handleSubmitUri = (uri) => {
         props.getMappingArtifactByMapName();
-        props.fetchSrcDocFromUri(uri);
+        props.fetchSrcDocFromUri(uri,props.mapIndex);
```

Comment:
Can you pls explain what this prop is for -- props.mapIndex. Should this be added to mapProps in data.config?

Reply:
It wouldn't affect any of the existing tests. I ran them locally and made sure they are working fine. But yeah when we will write more tests using the function props.fetchSrcDocFromUri, then yes we would have to add that in the data.config.